modules/pytket-iqm/setup.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020-2022 Cambridge Quantum Computing
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import shutil
+import os
+from setuptools import setup, find_namespace_packages  # type: ignore
+
+metadata: dict = {}
+with open("_metadata.py") as fp:
+    exec(fp.read(), metadata)
+shutil.copy(
+    "_metadata.py",
+    os.path.join("pytket", "extensions", "iqm", "_metadata.py"),
+)
+
+setup(
+    name=metadata["__extension_name__"],
+    version=metadata["__extension_version__"],
+    author="Alec Edgington",
+    author_email="alec.edgington@cambridgequantum.com",
+    python_requires=">=3.9",
+    url="https://github.com/CQCL/pytket-extensions",
+    description="Extension for pytket, providing access to IQM backends",
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
+    license="Apache 2",
+    packages=find_namespace_packages(include=["pytket.*"]),
+    include_package_data=True,
+    install_requires=["pytket ~= 0.18.0", "iqm-client ~= 1.6"],
```

Comment:
Are you sure you want to use 1.7 here if available? maybe `==1.6` would be the better choice? They release a new version every month. (https://pypi.org/project/iqm-client/#history)

Reply:
( `~=1.6.0` would maybe work too, not sure.)

Reply:
If they are following semantic versioning conventions then 1.7 should be API-compatible with 1.6. (If not we can revisit this, but probably better to use the latest 1.x version available.)